scorescreen.py

```diff
@@ -83,7 +83,7 @@ def make(self, gameDisplay, back, score, fruitscore):
         back.set_alpha(225)
 
         font_path = "fonts/Arimo.ttf"
-        font_size = 55
+        font_size = 50
```

Comment:
Why is this changed?

Reply:
Sir I dint change it. It was done before